tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,94 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// Lets say we have component with name 'self-clear-cache.hbs'
+// The UI Design can be found here : https://www.figma.com/file/LMujWCqnzXl1wQJVwAzCPV/Clear-your-member-details-page?node-id=0%3A1
+// It shows the time at which user clears the cache last time.
+// Last Requested : {{ time }}
+
+module('Cache Clear Page', (hooks) => {
+  setupRenderingTest(hooks);
+
+  test('should show last time user cleared the cache', async function (assert) {
+    assert.expect(1);
+
+    // set the outer context '23 March 1:23 pm IST'
+    this.set('lastTime', '23 March 1:23 pm IST');
+
+    await render(hbs`<SelfClearCache @time={{this.lastTime}} />`);
+
+    assert.equal(
+      this.element.querySelector('.last-time').textContent,
```

Comment:
Rather than using `querySelector` can we use Ember Test Selectors?

Docs here: https://guides.emberjs.com/release/testing/#toc_ember-test-selectors

![image](https://user-images.githubusercontent.com/56217868/114543462-dd6e4f00-9c76-11eb-9757-34c167b879ce.png)

Reply:
Great catch @swarajpure :)